UnitsNet.Serialization.JsonNet/UnitsNet.Serialization.JsonNet.csproj

```diff
@@ -23,7 +23,7 @@
     <AssemblyVersion>4.0.0.0</AssemblyVersion> <!-- Should reflect major part of Version -->
     <LangVersion>latest</LangVersion>
     <RootNamespace>UnitsNet.Serialization.JsonNet</RootNamespace>
-    <TargetFrameworks>netstandard2.0;net40</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net40;net47</TargetFrameworks>
```

Comment:
It seems you maybe missed this one. Changing target framework is out of scope for this PR.

Please review the final diff of the PR in the `Files Changed` tab and revert any changes that are not related to adding this unit. Whitespace changes are fine.

https://github.com/angularsen/UnitsNet/pull/978/files